.github/workflows/mkdocs.yml

```diff
@@ -0,0 +1,36 @@
+# This GitHub Actions workflow contains two jobs:
+# -- First, Lint uses the latest Ubuntu image and Python 3.9 to check
+# the code and writing for defects, using the Poetry lint task
+#
+# -- Second, Test uses a strategy matrix to test the code with different
+# Python versions and determine code coverage, using the Poetry test task
```

Comment:
This isn't an accurate comment.

Reply:
Hello @Michionlion could you clarify what is not accurate about this comment?

Reply:
Sure! This specific file (`mkdocs.yml`) doesn't contain a `Lint` or `Test` job; in fact, it contains only a `mkdocs` job, and the name of the workflow (on line 8 below) is also misleading. Hopefully that gives you enough to go on to correct some of this documentation and naming!

Reply:
Updated comments to be accurate!